utils/src/main/scala/za/co/absa/enceladus/utils/testUtils/SparkTestBase.scala

```diff
@@ -17,17 +17,69 @@ package za.co.absa.enceladus.utils.testUtils
 
 import org.apache.log4j.{Level, Logger}
 import org.apache.spark.sql.SparkSession
+import org.apache.spark.SparkConf
+import com.typesafe.config.ConfigFactory
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{Path, FileSystem}
+import scala.collection.JavaConversions._
+import java.io.File
 import za.co.absa.enceladus.utils.time.TimeZoneNormalizer
 
-trait SparkTestBase {
+trait SparkTestBase { self =>
   System.setProperty("user.timezone", "UTC");
 
-  implicit val spark = SparkSession.builder().master("local[*]").appName("test")
-  .config("spark.sql.codegen.wholeStage", false)
-  .config("spark.ui.enabled", "false")
-  .config("spark.driver.bindAddress","127.0.0.1")
-  .config("spark.driver.host", "127.0.0.1")
-  .getOrCreate()
+  val config = ConfigFactory.load()
+  val sparkMaster = config.getString("enceladus.utils.testUtils.sparkTestBaseMaster")
+
+  val sparkBuilder = SparkSession.builder()
```

Comment:
`config`, `sparkMaster`, `sparkBuilder` - Type annotations missings. I think we agreed on all public attributes to have them some time ago. I might be wrong.

Or make it private?